web/src/containers/activity/ContractorResource/ContractorResourceReview.js

```diff
@@ -46,11 +46,27 @@ const ContractorResourceReview = ({
         <li>
           <strong>Total Contract Cost:</strong> <Dollars>{totalCost}</Dollars>
         </li>
-        {apdFFYs.map(ffy => (
-          <li key={ffy}>
-            <strong>FFY {ffy} Cost:</strong> <Dollars>{years[ffy]}</Dollars>
-          </li>
-        ))}
+        {apdFFYs.map(ffy => {
```

Comment:
a shortcut you could do here is to do 
```
{apdFFYs.map(ffy => hourly.useHourly === false ? 
```
when you create an unnamed function it automatically returns